Realm/Realm/DatabaseTypes/Metadata.cs

```diff
@@ -42,5 +42,15 @@ public Metadata(TableKey tableKey, IRealmObjectHelper helper, IDictionary<string
             PropertyIndices = new ReadOnlyDictionary<string, IntPtr>(propertyIndices);
             Schema = schema;
         }
+
+        public static IntPtr GetPropertyIndex(string propertyName, Metadata metadata)
+        {
+            if (metadata.PropertyIndices.TryGetValue(propertyName, out var result))
+            {
+                return result;
+            }
+
+            throw new MissingMemberException(metadata.Schema.Name, propertyName);
```

Comment:
Why did you move this?

Reply:
After talking with @fealebenpae we concluded that it made more sense to have the method in the Metadata itself. Please reach out if you disagree. (This might also add some noise to the PR itself as well..)